mtools/util/logevent.py

```diff
@@ -560,6 +565,42 @@ def ninserted(self):
 
         return self._ninserted
 
+    @property
+    def bytesRead(self):
+        """Extract bytesRead counter if available (lazy)."""
+        if not self._counters_calculated:
+            self._counters_calculated = True
+            self._extract_counters()
+
+        return self._bytesRead
+
+    @property
+    def bytesWritten(self):
+        """Extract bytesWritten counter if available (lazy)."""
+        if not self._counters_calculated:
```

Comment:
might be better to factor out this self._counters_calculated block to a "private" function then just call it on the properties

(also I would change the order and set self._counters_calculated *after* self._extract_counters ran 😉 )